src/openeo_gfmap/stac/constants.py

```diff
@@ -0,0 +1,232 @@
+    "https://stac-extensions.github.io/processing/v1.1.0/schema.json",
+    "https://stac-extensions.github.io/projection/v1.1.0/schema.json",
+]
+CONSTELLATION = ["sentinel-2"]
+PLATFORM = ["sentinel-2a", "sentinel-2b"]
+INSTRUMENTS = ["msi"]
+GSD = [10, 20, 60]
+SUMMARIES = pystac.Summaries(
+    {
+        "constellation": CONSTELLATION,
+        "platform": PLATFORM,
+        "instruments": INSTRUMENTS,
+        "gsd": GSD,
+    }
+)
+
+
+def create_spatial_dimension(
+    name: str,
+) -> pystac.extensions.datacube.HorizontalSpatialDimension:
+    return pystac.extensions.datacube.HorizontalSpatialDimension(
+        {
+            "axis": name,
+            "step": 10,
+            "reference_system": {
+                "$schema": "https://proj.org/schemas/v0.2/projjson.schema.json",
+                "area": "World",
+                "bbox": {
+                    "east_longitude": 180,
+                    "north_latitude": 90,
+                    "south_latitude": -90,
+                    "west_longitude": -180,
+                },
+                "coordinate_system": {
+                    "axis": [
+                        {
+                            "abbreviation": "Lat",
+                            "direction": "north",
+                            "name": "Geodetic latitude",
+                            "unit": "degree",
+                        },
+                        {
+                            "abbreviation": "Lon",
+                            "direction": "east",
+                            "name": "Geodetic longitude",
+                            "unit": "degree",
+                        },
+                    ],
+                    "subtype": "ellipsoidal",
+                },
+                "datum": {
+                    "ellipsoid": {
+                        "inverse_flattening": 298.257223563,
+                        "name": "WGS 84",
+                        "semi_major_axis": 6378137,
+                    },
+                    "name": "World Geodetic System 1984",
+                    "type": "GeodeticReferenceFrame",
+                },
+                "id": {"authority": "OGC", "code": "Auto42001", "version": "1.3"},
+                "name": "AUTO 42001 (Universal Transverse Mercator)",
+                "type": "GeodeticCRS",
+            },
+        }
+    )
+
+
+TEMPORAL_DIMENSION = pystac.extensions.datacube.TemporalDimension(
+    {"extent": ["2015-06-23T00:00:00Z", "2019-07-10T13:44:56Z"], "step": "P5D"}
+)
+
+BANDS_DIMENSION = pystac.extensions.datacube.AdditionalDimension(
+    {
+        "values": [
+            "S2-SCL",
+            "S2-B01",
+            "S2-B02",
+            "S2-B03",
+            "S2-B04",
+            "S2-B05",
+            "S2-B06",
+            "S2-B07",
+            "S2-B08",
+            "S2-B8A",
+            "S2-B09",
+            "S2-B10",
+            "S2-B11",
+            "S2-B12",
+            "CROPTYPE",
```

Comment:
I renamed it to LABEL and will content myself with it. I think it's about time to merge this huge branch. Let's talk on monday about this because I'm not totally sure either @VincentVerelst  took care of that part